app/controllers/barbeque/sns_subscriptions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Barbeque::SnsSubscriptionsController < Barbeque::ApplicationController
+  def index
+    @sns_subscriptions = Barbeque::SNSSubscription.all
+  end
+
+  def show
+    @sns_subscription = Barbeque::SNSSubscription.find(params[:id])
+  end
+
+  def new
+    @sns_subscription = Barbeque::SNSSubscription.new
+  end
+
+  def edit
+    @sns_subscription = Barbeque::SNSSubscription.find(params[:id])
+  end
+
+  def create
+    @sns_subscription = Barbeque::SNSSubscription.new(params.require(:sns_subscription).permit(:topic_arn, :job_queue_id, :job_definition_id))
+
+    if @sns_subscription.save
```

Comment:
In the later commits I add a feature to subscribe/unsubscribe topic when SNSSubscription is created/deleted.
How about it?